tests/unit/apps/utils.py

```diff
@@ -18,3 +18,12 @@ def assert_plan_description(upgrade_plan, steps_description):
     sub_steps_check = zip(upgrade_plan.sub_steps, steps_description)
     for sub_step, description in sub_steps_check:
         assert sub_step.description == description
+
+
+def assert_plan(upgrade_plan, expected_plan):
+    assert len(upgrade_plan.sub_steps) == len(expected_plan)
+    sub_steps_check = zip(upgrade_plan.sub_steps, expected_plan)
+    for sub_step, expected_step in sub_steps_check:
+        assert sub_step.description == expected_step["description"]
+        assert expected_step["function_name"] in sub_step.function.__dict__.values()
+        assert sub_step.params == expected_step["params"]
```

Comment:
nit: it would be more readable to keep the same L/R order between actual and expected. Since the `in` keyword locks you into keeping the expected part on the L I would flip the first and third assertions.